lib/doorkeeper/rails/helpers.rb

```diff
@@ -11,11 +11,11 @@ def doorkeeper_authorize!(*scopes)
         end
       end
 
-      def doorkeeper_unauthorized_render_options
+      def doorkeeper_unauthorized_render_options(_error:)
```

Comment:
I'm sorry, we don't need the underscore here, makes no sense to require such a key. Can you please remove it?